<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>logging</class>
 <widget class="QWidget" name="logging">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Logging</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,1">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLineEdit" name="le_logfile">
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="placeholderText">
          <string>Log File</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_logfile">
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="btn_start">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_stop">
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="splitter_outlog">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="QSplitter" name="splitter_logitems">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Trace Events:</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QLineEdit" name="le_search">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_search">
              <property name="maximumSize">
               <size>
                <width>80</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Search</string>
              </property>
              <property name="iconSize">
               <size>
                <width>16</width>
                <height>16</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="btn_expand">
              <property name="maximumSize">
               <size>
                <width>30</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>▼</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_collapse">
              <property name="maximumSize">
               <size>
                <width>30</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>▲</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTreeWidget" name="tree_tevents">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
          <property name="headerHidden">
           <bool>true</bool>
          </property>
          <column>
           <property name="text">
            <string>Trace Events</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Log Masks:</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="minimumSize">
           <size>
            <width>125</width>
            <height>0</height>
           </size>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scroll_contents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>405</width>
             <height>184</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="spacing">
             <number>0</number>
            </property>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QPlainTextEdit" name="out_log">
      <property name="font">
       <font>
        <family>Monospace</family>
       </font>
      </property>
      <property name="lineWrapMode">
       <enum>QPlainTextEdit::NoWrap</enum>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="maximumBlockCount">
       <number>1000</number>
      </property>
      <property name="centerOnScroll">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
